src/Theme.js

```diff
@@ -0,0 +1,39 @@
+import { createTheme } from "@mui/material/styles";
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: "#212121",
```

Comment:
make yellow color is the primary in the Palette , and black is the secondary

Reply:
it might take time i will check it later